server/src/main/java/com/broadcom/lsp/cobol/LangServerBootstrap.java

```diff
@@ -96,8 +96,9 @@ boolean isPipeEnabled(@NonNull String[] args) {
       LOG.info("Language server started using socket communication on port [{}]", LSP_PORT);
       LOG.info("Java version: " + Runtime.version());
       // wait for clients to connect
-      Socket socket = serverSocket.accept();
-      return createServerLauncher(server, socket.getInputStream(), socket.getOutputStream());
+      try (Socket socket = serverSocket.accept()) {
```

Comment:
<img width="842" alt="LangServerBootstrap" src="https://user-images.githubusercontent.com/10369965/98956863-d644e880-2500-11eb-8ca2-6180173c7117.png">

Reply:
```

INFO: Socket closed
java.net.SocketException: Socket closed
	at java.base/sun.nio.ch.NioSocketImpl.ensureOpenAndConnected(NioSocketImpl.java:166)
	at java.base/sun.nio.ch.NioSocketImpl.beginRead(NioSocketImpl.java:232)
	at java.base/sun.nio.ch.NioSocketImpl.implRead(NioSocketImpl.java:300)
	at java.base/sun.nio.ch.NioSocketImpl.read(NioSocketImpl.java:351)
	at java.base/sun.nio.ch.NioSocketImpl$1.read(NioSocketImpl.java:802)
	at java.base/java.net.Socket$SocketInputStream.read(Socket.java:937)
	at java.base/java.net.Socket$SocketInputStream.read(Socket.java:932)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.listen(StreamMessageProducer.java:79)
	at org.eclipse.lsp4j.jsonrpc.json.ConcurrentMessageProcessor.run(ConcurrentMessageProcessor.java:99)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:830)

```

Reply:
Have checked the first time without client-side and missed this error, thanks!
Reverted.